src/main/scala/com/gd/twitteranalytics/TwitterReportGenerator.scala

```diff
@@ -0,0 +1,11 @@
+package com.gd.twitteranalytics
+
+import org.apache.spark.sql.{DataFrame,SparkSession}
+
+object TwitterReportGenerator {
+
+  def execute(f: (SparkSession, DataFrame) => DataFrame, spark: SparkSession, tweetStatusDf: DataFrame) = f(spark, tweetStatusDf)
```

Comment:
why there are two `execute` functions?
it even causes compilation error. 
Please run tests before committing and pushing to a remote branch.

Reply:
Is it showing two execute functions in your copy ? At branch, I can see only one..

Reply:
well.. this is form the branch: https://github.com/ipetrushin-gd/data-engineering-practice/blob/dbbb4bb45e9a6f39a51e64f860d555aeba5e6aed/src/main/scala/com/gd/twitteranalytics/TwitterReportGenerator.scala

Reply:
Wow !!
Sorry, this is some git glitch.. because it would clearly show me a compilation
 error if I would try to commit file with two same methods.
Fixing it rite away..